DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: urltools
 Type: Package
 Title: Vectorised Tools for URL Handling and Parsing
-Version: 1.5.1
+Version: 1.5.1.9000
```

Comment:
Sure, I was just leaving like that while you decided the versioning. I'd vote for 1.5.2, because no major changes after 1.5.0, just bug fixes